src/lib/fees/parameters/thorchainLp.ts

```diff
@@ -1,5 +1,6 @@
 import type { FeeCurveParameters } from './types'
 
+// THOR asset doesn't have any fee models for now as it's not using curves, we only zero out values to make TS happy with less risks
```

Comment:
comment on wrop fee param

Reply:
Done in [05daabc](https://github.com/shapeshift/web/pull/8044/commits/05daabc89d2082b788ce937e871349e89d2193ba)